app/main.js

```diff
@@ -94,8 +94,7 @@ ipcMain.on('updateConfig', () => {
 
 ipcMain.on('getFolderLocations', event => {
   event.returnValue = {
-    settings: app.getPath('userData'),
-    plugins: path.join(path.dirname(app.getPath('exe')), 'plugins')
```

Comment:
Background on this - the only code that uses this uses the settings directory. I thought it would be simpler to remove plugins from here than support an array of folders.

Reply:
Sure, I don't mind. This is a relic from old times in case we need it somehow. Some people point their files folder to a cloud folder sometimes and they might want to config the plugins folder separately, but I think that's something for future versions if even.